### ISSUE TITLE
爱可可微博热门分享(10.29)

### ISSUE THREAD

**Issue by fly51fly** [fly51fly/aicoco#314]
No 1. <a href="https://weibo.com/1402400261/ODDdQy2J0">数字人模型：能在手机上实时运行的超轻量级虚拟人，适合移动端的实时互动和娱乐</a>
No 2. <a href="https://weibo.com/1402400261/ODFY9AmZY">x.infer：框架无关的计算机视觉推理库，它允许用户通过统一的 Python API 运行来自不同框架的多种模型的推理，支持超过1000+模型</a>
No 3. <a href="https://weibo.com/1402400261/Npk9D6zcR">过节了，推荐一些微博上的程序员大佬。排名不分先后，简介如有错误欢迎指出。也欢迎评论增加你知道的程序员@刘群MT-to-Death ......</a>
No 4. <a href="https://weibo.com/1402400261/ODG2rzava">mdx - 执行Markdown中的代码块，让你能直接在Markdown文件中编写并执行代码块，支持多种编程语言，使得文档和代码的结合更加紧密</a>
No 5. <a href="https://weibo.com/1402400261/ODDf6wqSx">NewsNow：优雅阅读实时热门新闻的工具，提供简洁的界面和流畅的阅读体验，让你随时掌握最新资讯</a>
No 6. <a href="https://weibo.com/1402400261/ODJ0o8wx1">早！[太阳] ​</a>
No 7. <a href="https://weibo.com/1402400261/ODJ9L0on1">[LG]《Newton Losses: Using Curvature Information for Learning with Differentiable Algorithms》</a>
No 8. <a href="https://weibo.com/1402400261/ODJeaiXto">[LG]《Mixture of Parrots: Experts improve memorization more than reasoning》</a>
No 9. <a href="https://weibo.com/1402400261/ODqYxhzUd">#免费##抽奖# 携手@博文视点Broadview 送出3本《事理图谱：概念与技术》</a>
No 10. <a href="https://weibo.com/1402400261/ODDeIgx6p">ngrid：数据浏览好帮手，就像命令行的“less”命令一样，让你在命令行环境下轻松浏览大型表格数据</a>
No 11. <a href="https://weibo.com/1402400261/ODJh6mx0J">[LG]《Computational Bottlenecks of Training Small-scale Large Language Models》</a>
No 12. <a href="https://weibo.com/1402400261/ODG4Z6ke9">几篇论文实现代码</a>
No 13. <a href="https://weibo.com/1402400261/ODJebENmt">混合专家模型在知识密集型任务中展现出卓越的记忆能力，但在推理任务方面却不如密集 Transformer，这揭示了模型架构与任务类型之间存在深刻的联系，并非所有任务都适合 ......</a>
No 14. <a href="https://weibo.com/1402400261/ODzQUFCo0">[CL] A Survey of Conversational Search 这篇论文全面综述了对话式搜索技术，重点分析了其四个关键模块以及大型语言模型的整合，并指出了该领域面临的挑战和未来的发展方向，尤其强调了多轮交互的复杂性以及如何平衡检索和生成模块以实现更精准、更符合用户需求的搜索体 ......</a>
No 15. <a href="https://weibo.com/1402400261/ODG3yxVrC">EngineAI Legged Gym：一个用于训练机器人在复杂地形上行走的仿真环境，集成了模拟到现实转移所需的所有组件，包括执行器网络、摩擦与质量随机化、带噪声的观测和训练中的随机推动</a>
No 16. <a href="https://weibo.com/1402400261/ODqUfohUJ">[LG] Long Term Memory: The Foundation of AI Self-Evolution 本文提出AI应具备自我进化能力，并以长期记忆(LTM)为核心机制，构建多智能体框架OMNE，在GAIA基准测试中取得领先，为AI持续学习和个性化发展提供了新的方向。 ......</a>
No 17. <a href="https://weibo.com/1402400261/ODJ7nw16p">[LG]《Why do Random Forests Work? Understanding Tree Ensembles as Self-Regularizing Adaptive Smoothers》</a>
No 18. <a href="https://weibo.com/1402400261/ODJk9cSnB">今日推介(第1573期)：通过概率可发现提取来测量记忆、将树集成理解为自正则化自适应平滑器、利用二阶信息改进损失函数、混合专家提高的更多是记忆而非推理、训练小规模大型语言模型的计算瓶颈 ......</a>
No 19. <a href="https://weibo.com/1402400261/ODGwa7Q28">晚安～ [月亮] ​</a>
No 20. <a href="https://weibo.com/1402400261/ODdSrnbf7">DataExpert 数据工程：一个公共知识库，涵盖所有由大型语言模型（LLM）驱动的数据工程概念，帮助你理解并应用LLM技术于数据工程领域</a>
No 21. <a href="https://weibo.com/1402400261/ODzHBtiAX">[LG]《Asynchronous RLHF: Faster and More Efficient Off-Policy RL for Language Models》</a>
No 22. <a href="https://weibo.com/1402400261/ODD3Rd7FH">GenoView：基于raylib的动画查看应用，专门用来展示Geno角色的动画，支持骨骼动画数据的清晰展示，便于发现角色动画中的常见问题</a>
No 23. <a href="https://weibo.com/1402400261/ODJh8tUhJ">本文通过系统研究不同硬件配置和训练策略对小型语言模型训练成本效率的影响，发现Flash Attention对小型模型尤为重要，并提出了一种新的成本效益评估指标“每美元Token数”，为低资源环境下的高效SLM训练提供了实用指导。</a>
No 24. <a href="https://weibo.com/1402400261/ODzDooc9p">[LG]《Beyond position: how rotary embeddings shape representations and memory in autoregressive transfomers》</a>
No 25. <a href="https://weibo.com/1402400261/ODJyWwVAf">//@爱可可-爱生活：欢迎参与～</a>
No 26. <a href="https://weibo.com/1402400261/ODmit2mCD">Computer Use Agents资源库：一个收集了关于Computer Use Agents的资源库，包括论文和博客，旨在帮助理解和开发能够像人类一样使用计算机的智能代理</a>
No 27. <a href="https://weibo.com/1402400261/OD1g4abv8">Agent2Sim：从日常视频中学习交互行为的智能模拟系统，能够模拟和学习人与环境的互动，用于研究和开发更自然的交互体验</a>
No 28. <a href="https://weibo.com/1402400261/OCIM9tXpj">CUDATutorial：手把手教你CUDA高性能编程，从入门到精通，一起学习一起进步</a>
No 29. <a href="https://weibo.com/1402400261/ODJbDhWcp">本文提出了一种新的 Newton 损失方法，通过巧妙地将二阶优化应用于损失函数本身，有效解决了弱监督学习中可微算法带来的非凸优化难题，在多个基准测试中取得了显著的性能提升，并展现了其在计算效率和泛化能力方面的优势。//[LG]《Newton Losses: Using Curvature Information for Learning with Differentiable Algorithms》</a>
No 30. <a href="https://weibo.com/1402400261/OD4asdMpZ">Agent-0：一个旨在模拟OpenAI新发布的O1模型推理能力的项目，通过思维链和迭代推理来改进问题解决方案</a>
No 31. Luxirty Search：一款基于Google的搜索引擎，主打无广告、无跟踪，提供干净、简洁、快速的搜索体验，特别屏蔽了内容农场，保护用户远离低质内容
No 32. <a href="https://weibo.com/1402400261/ODzPI7nJn">[LG]《Explaining Graph Neural Networks with Large Language Models: A Counterfactual Perspective for Molecular Property Prediction》</a>
No 33. LOKI：大型多模态模型的综合性合成数据检测基准，用于评估不同数据格式和超过25种主流多模态模型的性能
No 34. <a href="https://weibo.com/1402400261/OD4bJ6wc8">Parrot-TTS：基于 Transformer 的序列到序列文本到语音（TTS）系统，使用 HuBERT 量化单元和修改版 HiFi-GAN 声码器进行语音合成</a>
No 35. <a href="https://weibo.com/1402400261/OCh7rCHCM">学术项目页面模板：为你的研究论文创建一个简洁的项目页面，支持响应式布局，易于编辑，且对SEO友好，特色包括LaTeX公式、代码块、视频嵌入等</a>
No 36. [LG]《Losing dimensions: Geometric memorization in generative diffusion》
No 37. <a href="https://weibo.com/1402400261/OBirWhJ20">[LG]《RL, but don't do anything I wouldn't do》</a>
No 38. <a href="https://weibo.com/1402400261/OCRDXclSm">OURS Project：开源、可升级、可修复、无大科技公司控制的智能手机项目，提供详细的组装指南，让你能自制一部完全属于自己的智能手机</a>
No 39. <a href="https://weibo.com/1402400261/OCyZ8dO6n">DocLayout-YOLO：文档布局分析工具，通过多样化的合成数据和全局到局部的自适应感知增强文档布局分析，提升处理速度和准确性</a>
No 40. AI扩图网站：开源的 AI 扩图网站源码仓库，提供了基于 Next.js 和 Tailwind CSS 的网站代码模板，支持多语言国际化，集成了 Google 登录和支付功能，以及与 AI 模型的交互
No 41. <a href="https://weibo.com/1402400261/Ne1NEqJyA">今日推介(第1128期)：基于用户成对偏好的优化算法、语言模型生成的批判模型、通过分阶段推测解码加速LLVM推理、通过对齐语言将大型语言模型推广到非英语、基于全方位扩散模型的文本引导通用音乐生成 ......</a>
No 42. <a href="https://weibo.com/1402400261/ODdRYxlxk">jax-flash-attn2：一个为JAX设计的灵活高效的Flash Attention 2.0实现，支持GPU/TPU/CPU多后端和Triton/Pallas/JAX多平台</a>
No 43. 《LLM工程师手册》随书代码：一本深入浅出的指南，从基础到在AWS上部署高级LLM和RAG应用程序，使用LLMOps最佳实践
No 44. <a href="https://weibo.com/1402400261/ODPhGbsJs">[TAI快报]Vol.119 记忆与智能】本期「TAI快报」，带你深入了解AI领域的五大前沿研究。从大型语言模型的记忆能力，到随机森林的团队合作原理，再到高效优化的Newton损失方法，混合专家模型的记忆与推理平衡，以及训练小规模AI模型的经济秘诀。小爱和小T用通俗易懂的语言，带你揭秘AI背后的智慧与挑战。 ......</a>
No 45. <a href="https://weibo.com/1402400261/ODgRAc3il">[LG]《Rethinking Softmax: Self-Attention with Polynomial Activations》</a>
No 46. <a href="https://weibo.com/1402400261/ODdEZB07Z">具身智能入门指南：为新手提供快速建立关于具身智能领域认知的路径和有用信息，帮助理解智能体与环境的交互获取信息、理解问题、做出决策并实现行动的过程</a>
No 47. <a href="https://weibo.com/1402400261/ODzKrjxQv">[CL]《Little Giants: Synthesizing High-Quality Embedding Data at Scale》</a>
No 48. <a href="https://weibo.com/1402400261/ODn6Rm09b">Hollama：一个简洁的网页界面，用于与Ollama服务器对话，支持大型提示字段、Markdown渲染、代码编辑功能，以及自定义系统提示和高级Ollama参数</a>
No 49. <a href="https://weibo.com/1402400261/ODqGov9NC">[LG]《Automatically Interpreting Millions of Features in Large Language Models》</a>
No 50. <a href="https://weibo.com/1402400261/ODqKb0RLI">[CL]《A Complexity-Based Theory of Compositionality》</a>
